### PR TITLE
Fixes issue #7825 Unable to search in open files if assembly browser

### PR DESCRIPTION
**Pull request by mkrueger**
open

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/Scope.cs [hunk 1]
```diff
@@ -276,6 +276,8 @@ public override IEnumerable<FileProvider> GetFiles (ProgressMonitor monitor, Fil
 				if (textBuffer != null) {
 					yield return new OpenFileProvider (textBuffer, document.Owner as Project, document.FileName);
 				} else {
+					if (document.IsViewOnly || document.FileName.IsNullOrEmpty)
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/Scope.cs]: You could just check document.IsFile

Comment (mkrueger) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/Scope.cs]: Does that work for untitled files then ? I don't think I can safe the file name check here.

**Comment by sevoku** [comment]
@monojenkins backport release-8.1

**Pull request merged**